composer.json

```diff
@@ -10,7 +10,7 @@
         "ext-pdo": "*",
         "dedoc/scramble": "^0.9.0",
         "guzzlehttp/guzzle": "^7.7",
-        "inertiajs/inertia-laravel": "^0.6.9",
+        "inertiajs/inertia-laravel": "^1.0.0",
```

Comment:
Do we really need Inertia.js in project with API?

Reply:
There's still web application part left, should I add issue for cleaning up unnecessary parts, and keep only API?

Reply:
Yes